server/src/main/java/org/elasticsearch/search/aggregations/support/MultiValuesSourceFieldConfig.java

```diff
@@ -135,28 +157,22 @@ protected MultiValuesSourceFieldConfig(
         this.filter = filter;
         this.userValueTypeHint = userValueTypeHint;
         this.format = format;
+        this.includeExclude = includeExclude;
     }
 
     public MultiValuesSourceFieldConfig(StreamInput in) throws IOException {
-        if (in.getVersion().onOrAfter(Version.V_7_6_0)) {
```

Comment:
+1 to what is being said here. In any case removing these bwc checks should be removed via a separate change, so that we make a concussion decision about this.

Reply:
Reverted. Note regarding: "7.17 is still supported" - true, however at max `7.11` (EOL) serialization is changed. Anyway, your responsibility.